### PR TITLE
EMP-561-urgent-user-testing-back-button-behaviour

### PR DESCRIPTION
**Pull request by BNmoj**
Changes - 

- amended crmBackLink.ts
- amended crmBackLink.test.ts
- amended crm4Controller.test.ts
- amended crm5Controller.test.ts

### CODE REVIEW

File: server/utils/crmBackLink.ts [hunk 1]
```diff
@@ -1,23 +1,15 @@
 import { Request } from 'express'
 
 function manageBackLink(req: Request, currentUrl: string): string {
```
Comment (muyinatech) [inline server/utils/crmBackLink.ts:3]:
```suggestion
function manageBackLink(currentUrl: string): string {
```

**Comment by muyinatech** [comment]
fromBack query parameter is no longer used. So this line in SearchEformController.ts be changed from:

`      if (!req.query.page || req.query.fromBack) {`

to:
`      if (!req.query.page) {`

File: server/utils/crmBackLink.ts (hunk 1 above)
Comment (BNmoj) [reply server/utils/crmBackLink.ts:3]: this needs to be left in otherwise it will trigger a currentUrl.includes is not a function error

Comment (muyinatech) [reply server/utils/crmBackLink.ts:3]: All calls like so `manageBackLink(req, currentUrl)` need to be changed to `manageBackLink(currentUrl)`

Comment (BNmoj) [reply server/utils/crmBackLink.ts]: done

**Review by muyinatech — APPROVED** [review]

**Pull request merged**